core/twip/generic_task.py

```diff
@@ -240,16 +262,15 @@ def compute_rewards_twip(
     # rewards *= 1.0 + time_factor
 
     # Harsh penalty for falling
-    fall_penalty = torch.where(torch.abs(roll) > 0.5, -5.0, 0.0)
-    rewards += fall_penalty
+    rewards = torch.where(torch.abs(roll) > 0.5, -2.0, rewards)
```

Comment:
0.5? wouldn't it be 3.4(based on the real world testing)

Reply:
3.4rad?

Reply:
yup. not sure about the .4 but it's  definitely 3

Reply:
It should fail after 3 radians? (a little less than 180 degrees)

Reply:
sorry i mean 0.34 @AugustDG

Reply:
Ahh, in this case it's when it falls off the terrain that it would trigger: we removed the penalty for touching the floor a while back